adafruit_stmpe610.py

```diff
@@ -240,8 +318,53 @@ def __init__(self, i2c, address=_STMPE_ADDR):
             raise RuntimeError("Failed to find STMPE610! Chip Version 0x%x" % version)
         super().__init__()
 
+    @property
+    def touch_point(self):  # pylint: disable=too-many-branches
```

Comment:
Why is touch_point replicated for both the I2C and SPI implementations .
Couldn't this be the same for both like the other properties?

Reply:
The first submitted version placed touch_point only in the parent class, but pylint didn't like it. It didn't recognize screen_size, etc. as members of the parent class. At least the touch_point duplicated version will be easy to change if the I2C class is removed.